www/pages/project-advanced.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+permalink: /project-advanced
+title: Transfer or delete a project
+---
+
+To access the advanced project settings:
+
+- Select **SETTINGS** from above the project files
+- At the bottom of the page, you have three options:
+  - Public/private setting: to make your project public or private
+  - **Transfer project**: this option transfers the ownership of your project to another Mergin user or organisation.
```

Comment:
This needs a bit more detail: once you choose to transfer a project, the user/org needs to accept that request in the web GUI (where?). Also, they get email notification. The transfer request expires after ? days if not accepted.